app/controllers/api/deploy_environments_controller.rb

```diff
@@ -41,7 +41,11 @@ def scale
   end
 
   def index
-    render status: 200, json: current_user.publishers.find(params['publisher_id']).deploy_environments
+    if(params['publisher_id'])
+        render status: 200, json: current_user.publishers.find(params['publisher_id']).deploy_environments
+    else
+        render status: 200, json: current_user.deploy_environments.order("created_at DESC").limit(30)
```

Comment:
@jkurambhatti Since the route itself has publisher_id, when will else condition be true?